<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdClass</class>
 <widget class="QWidget" name="ThresholdClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阈值设定</string>
  </property>
  <widget class="QLabel" name="label_low">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>下限阈值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_high">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上限阈值</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radio_inside">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保留范围内部</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_outside">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保留范围外部</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="ledit_low">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_high">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>195</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
